Decred Wallet/Features/More/More Features/Settings/SettingsController.swift

```diff
@@ -25,10 +26,12 @@ class SettingsController: UITableViewController  {
     @IBOutlet weak var spend_uncon_fund: UISwitch!
     @IBOutlet weak var beep_for_new_block: UISwitch!
     @IBOutlet weak var start_Pin: UISwitch!
+    @IBOutlet weak var use_biometric: UISwitch!
```

Comment:
No need to follow the variable naming pattern on this file, better to change all even to camelCasing.
```suggestion
    @IBOutlet weak var useBiometricSwitch: UISwitch!
```